charts/super-agent/Chart.yaml

```diff
@@ -25,8 +25,6 @@ keywords:
 maintainers:
   - name: sigilioso
     url: https://github.com/sigilioso
-  - name: gsanchezgavier
```

Comment:
:)

Reply:
I used the users from `coreint` instead of `caos`. I'll add all caos team. Sorry.

Reply:
Done in [#d598ded](https://github.com/newrelic/helm-charts/pull/1466/commits/d598dedf47d32be2ea9e055031faf859b75386fe)